### ISSUE TITLE
Teste funcional

### ISSUE THREAD

**Issue by luisfmsouza** [luisfmsouza/digite-uma-vez#8]
Protractor, behat ou outra opção

**Comment by andreroggeri** [comment]
@luisfmsouza seria o caso de integrar com o _Travis_ e o _Coveralls_ pra automatizar tudo isso

**Comment by luisfmsouza** [comment]
Sim! Exatamente isso, tanto no teste funcional quando o teste de código. 

Inclusive, acho que falta abrir a issue pra integrar com o Travis e Coveralls.
